Extension/CHANGELOG.md

```diff
@@ -342,7 +342,7 @@
 * Fix the default `cwd` for `cppbuild` tasks. [#6618](https://github.com/microsoft/vscode-cpptools/issues/6618)
 
 ## Version 1.1.2: November 17, 2020
-### Bug Fix
+### Bug Fixes
```

Comment:
I think this was intentionally "Bug Fix" since there's only 1 fix, but I don't mind either way.

Reply:
I thought the same thing, but there's already plenty of "Bug Fixes" with only 1 fix : 
https://github.com/microsoft/vscode-cpptools/blob/7a9d0aae08c84bb538603678060f26718992119c/Extension/CHANGELOG.md#L5-L6
https://github.com/microsoft/vscode-cpptools/blob/7a9d0aae08c84bb538603678060f26718992119c/Extension/CHANGELOG.md#L81-L82
https://github.com/microsoft/vscode-cpptools/blob/7a9d0aae08c84bb538603678060f26718992119c/Extension/CHANGELOG.md#L349-L350